autogen.sh

```diff
@@ -27,17 +27,22 @@
 #
 SUBMODULES="lib/ivykis lib/jsonc"
 GIT=`which git`
+# bootstrap syslog-ng itself
```

Comment:
I think this comment is not relevant here, we have this at line 85.

Reply:
Yes copy paste error :) I will remove it next time.